platform_it/pom.xml

```diff
@@ -495,6 +497,10 @@
                             <includes>
                                 <include>**/*IT.java</include>
                             </includes>
+                            <excludes>
+                                <exclude>com/splicemachine/derby/impl/sql/execute/upgrade/UpgradeTestIT.java
+                                </exclude>
```

Comment:
You can rename UpgradeTestIT to a name not ending with IT or Test, to avoid excluding it from IT